### PR TITLE
Polishing comments on helpers and adding extra line test data

### PR DESCRIPTION
**Pull request by Golodhros**
## Description
Polishing some helpers comments and variable names
Adding extra line chart test data

## Motivation and Context
General cleanup

## How Has This Been Tested?
Ran tests

## Types of changes

- [x] Refactor (changes the way we code something without changing its functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My code follows the [code style guide](https://github.com/eventbrite/britecharts/blob/master/CODESTYLEGUIDE.md) of this project.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/charts/helpers/style.js [hunk 1]
```diff
@@ -38,74 +38,74 @@ module.exports = (function() {
...
-            function serializeWithStyles(elem) {
-                let cssTexts = [],
-                    elements,
-                    computedStyle,
-                    defaultStyle,
-                    result;
+            cssTexts = [];
+            elements = elem.querySelectorAll('*');
+
+            [].forEach.call(elements, (el, i) => {
```
Comment (DalerAsrorov) [inline src/charts/helpers/style.js]: Why not use regular for loops or `forEach`?

File: src/charts/helpers/text.js [hunk 2]
```diff
@@ -71,76 +71,75 @@ define(function(require) {
...
-
-                while ((word = words.pop())) {
-                    line.push(word);
+                if (tspan.node() && tspan.node().getComputedTextLength() > width) {
+                    line.pop();
                     tspan.text(line.join(' '));
 
-                    if (tspan.node() && tspan.node().getComputedTextLength() > width) {
-                        line.pop();
+                    if (lineNumber < limit - 1) {
```
Comment (DalerAsrorov) [inline src/charts/helpers/text.js]: Shouldn't `(lineNumber < limit)` work instead? This looks like one of the lines will be left out.

File: src/charts/helpers/style.js (hunk 1 above)
Comment (Golodhros) [reply src/charts/helpers/style.js]: Good call. I am not sure why is this way.

I would love to refactor it a bit at some point, but for today I am just moving things around.

File: src/charts/helpers/text.js (hunk 2 above)
Comment (Golodhros) [reply src/charts/helpers/text.js]: Ha, no idea either!

I can add it to my list of stuff to investigate... but I know I will never get there!

**Comment by Golodhros** [comment]
Thanks for the quick review @DalerAsrorov!!

**Pull request merged**